app/src/main/kotlin/com/hippo/ehviewer/ui/screen/GalleryDetailScreen.kt

```diff
@@ -908,18 +907,12 @@ fun GalleryDetailScreen(args: GalleryDetailScreenArgs, navigator: DestinationsNa
         val startPage by rememberInVM {
             EhDB.getReadProgressFlow(galleryInfo.gid)
         }.collectAsState(0)
-        var readButtonText by rememberSaveable { mutableStateOf(readText) }
-        LifecycleResumeEffect(Unit) {
-            coroutineScope.launchIO {
-                runSuspendCatching {
-                    readButtonText = if (startPage == 0) {
-                        readText
-                    } else {
-                        context.getString(R.string.read_from, startPage + 1)
-                    }
-                }
+        val readButtonText = rememberSaveable(startPage) {
+            if (startPage == 0) {
+                readText
+            } else {
+                context.getString(R.string.read_from, startPage + 1)
```

Comment:
Now that startPage is a `MutableState`
Remove `rememberSaveable`, use `stringResource` instead

Reply:
Done